linkerd/app/inbound/src/http/mod.rs

```diff
@@ -270,7 +271,15 @@ pub mod fuzz_logic {
     pub use linkerd_app_test as support;
     use linkerd_app_test::*;
 
-    pub async fn fuzz_entry_raw(uri: &str, header_name: &str, header_value: &str, is_get: bool) {
+    #[derive(Debug, Arbitrary)]
+    pub struct TransportHeaderSpec {
```

Comment:
this probably should not be called `TransportHeaderSpec`, this is for testing the inbound HTTP proxy, not the linkerd transport header parser...

Reply:
Ah yes - thanks for catching it!